tests/src/test/java/org/eclipse/hono/tests/client/HonoKafkaConsumerIT.java

```diff
@@ -105,16 +106,17 @@ public static void init() {
         final Map<String, String> adminClientConfig = IntegrationTestSupport.getKafkaAdminClientConfig()
                 .getAdminClientConfig("test");
         adminClient = KafkaAdminClient.create(vertx, adminClientConfig);
-        final Map<String, String> producerConfig = IntegrationTestSupport.getKafkaProducerConfig()
-                .getProducerConfig("test");
-        producerConfig.put(ProducerConfig.MAX_REQUEST_SIZE_CONFIG, SMALL_TOPIC_SEGMENT_SIZE_BYTES);
-        producerConfig.put(ProducerConfig.BATCH_SIZE_CONFIG, SMALL_TOPIC_SEGMENT_SIZE_BYTES);
-        kafkaProducer = KafkaProducer.create(vertx, producerConfig);
     }
 
     @BeforeEach
     void printTestInfo(final TestInfo testInfo) {
         LOG.info("running test {}", testInfo.getDisplayName());
+        final Map<String, String> producerConfig = IntegrationTestSupport.getKafkaProducerConfig()
+                .getProducerConfig("test");
+        producerConfig.put(ProducerConfig.MAX_REQUEST_SIZE_CONFIG, "10000");
+        producerConfig.put(ProducerConfig.BATCH_SIZE_CONFIG, "6000");
+        producerConfig.put(ProducerConfig.METADATA_MAX_AGE_CONFIG, "300");
+        kafkaProducer = KafkaProducer.create(vertx, producerConfig);
```

Comment:
Now that the `kafkaProducer` is initialized in this method, it would be better to rename the method.